### ISSUE TITLE
LeNet

### ISSUE THREAD

**Issue by Algorithmist-Girl** [pytorch/vision#7996]
### 🚀 The feature

Hi, I'm interested in reproducing the neural network architecture "LeNet" from the paper "Gradient-Based Learning Applied to Document Recognition" (https://ieeexplore.ieee.org/document/726791)

### Motivation, pitch

My main motivation for contributing this neural network architecture is to have a simpler neural network architecture for users to utilize in any of their projects to test some initial hypothesis
It would be great if one of the Torchvision maintainers could let me know if this sounds good for contribution

**Comment by NicolasHug** [comment]
Hi @Algorithmist-Girl , thank you for the proposal.

I'm afraid LeNet is not really in scope for torchvision. While it has a strong historical significance, it is not used as a benchmark for research, or as a backbone, and its accuracy / perf will never be able to match architectures and techniques that came after it. As a result its usefulness is very limited.
